src/main/java/vazkii/botania/common/lib/LibMisc.java

```diff
@@ -18,7 +18,7 @@
 	public static final String MOD_ID = "Botania";
 	public static final String MOD_NAME = MOD_ID;
 	public static final String VERSION = "GRADLETOKEN_VERSION";
-	public static final String DEPENDENCIES = "required-after:Baubles;after:Thaumcraft";
+	public static final String DEPENDENCIES = "required-after:Baubles;after:Thaumcraft;after:NotEnoughItems@[2.2.7-GTNH,)";
```

Comment:
The patches made to Botania have been 100% agnostic of GTNH to allow other modpack builders to make use of these.

Hard dependencies on custom versions such as these are thus very much undesirable.